src/pages/getting-started/intermediates-tutorial.mdx

```diff
@@ -333,8 +337,10 @@ contract VRFD20 is VRFConsumerBaseV2 {
 ```
 
 <div class="remix-callout">
-  <a href="https://remix.ethereum.org/#url=https://docs.chain.link/samples/VRF/VRFD20.sol" target="_blank" >Open in Remix</a>
-  <a href="/getting-started/conceptual-overview/#what-is-remix" >What is Remix?</a>
+  <a href="https://remix.ethereum.org/#url=https://docs.chain.link/samples/VRF/VRFD20.sol" target="_blank">
+    Open in Remix
+  </a>
+  <a href="/getting-started/conceptual-overview/#what-is-remix">What is Remix?</a>
```

Comment:
This Remix button has CSS issues. This one might be fixed by putting the previous code sample into the code sample component, which has a Remix button by default.